src/plasmapy/__init__.py

```diff
@@ -26,7 +26,7 @@
 
 import sys
 
-if sys.version_info < (3, 9):  # coverage: ignore # noqa: UP036
+if sys.version_info < (3, 11):  # coverage: ignore
```

Comment:
Oops, forgot to update this line last year!  All so that we can make an infrequently needed `ImportError` message more descriptive...